### ISSUE TITLE
Copy/Duplicate/Export/Import Goals

### ISSUE THREAD

**Issue by atom-box** [piwik/piwik#18690]
Allow users with multiple sites to easily make goals on one site and then get those goals onto their other sites somehow. 

User: _Is there a possibility to create a Goal in one instance and then transfer it to other instances? We have Matomo on-premise in use to track all our 60 local website (worldwide). I now want to create the same "Standard" goals in each of those instances._

**Comment by lucaspal** [comment]
Is there any news about this enhancement?

**Comment by sgiehl** [comment]
Hi @lucaspal
There are not yet any plans on implementing that.
If you are able to provide a PR with the required changes we would be happy to review & merge it.

**Comment by lucaspal** [comment]
@sgiehl thanks for the quick reply. 

Is [this](https://developer.matomo.org/guides/getting-started-part-1) a good way to start getting an idea of the architecture of `matomo` or is there something else that I am missing?

**Comment by sgiehl** [comment]
@lucaspal Yes. The developer docs are a good start. If you have any questions feel free to get in touch on our forum: https://forum.matomo.org/
